`TestAccAppConfigDeployment_predefinedStrategy` failing during `ImportStateVerify`

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

#### Test output:
```
------- Stdout: -------
=== RUN   TestAccAppConfigDeployment_predefinedStrategy
=== PAUSE TestAccAppConfigDeployment_predefinedStrategy
=== CONT  TestAccAppConfigDeployment_predefinedStrategy
    deployment_test.go:65: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) (len=1) {
         (string) (len=5) "state": (string) (len=6) "BAKING"
        }
        
        
        (map[string]string) (len=1) {
         (string) (len=5) "state": (string) (len=9) "DEPLOYING"
        }
--- FAIL: TestAccAppConfigDeployment_predefinedStrategy (136.82s)
FAIL
```

### References

<!---
Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor documentation? For example:
--->

* #20288

--- a/internal/service/appconfig/deployment_test.go
+++ b/internal/service/appconfig/deployment_test.go
@@ -81,6 +81,11 @@ func TestAccAppConfigDeployment_predefinedStrategy(t *testing.T) {
 				ResourceName:      resourceName,
 				ImportState:       true,
 				ImportStateVerify: true,
+				// Since AppConfig Deployments can vary in completion times
+				// depending on the predefined deployment strategy,
+				// a waiter is not implemented for the resource;
+				// thus, we cannot guarantee the "state" value during import.
+				ImportStateVerifyIgnore: []string{"state"},
 			},
 		},
 	})

